phpunit.xml.dist

```diff
@@ -37,5 +37,8 @@ It's auto-generated by sonata-project/dev-kit package.
     <php>
         <ini name="precision" value="8" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
+        <env name="KERNEL_CLASS" value="\Sonata\PageBundle\Tests\App\AppKernel" />
+        <server name="APP_ENV" value="test" force="true" />
+        <server name="APP_DEBUG" value="false" />
```

Comment:
This file is auto generated ; do you have something in mind ?

Reply:
Yep, the same we generate the console file, we can generate the namespace for the AppKernel file.